mods/cnc/rules/structures.yaml

```diff
@@ -1199,7 +1199,7 @@ ANYPOWER:
 	AlwaysVisible:
 	Interactable:
 	Tooltip:
-		Name: Power Plant
+		Name: any power generation
```

Comment:
Can we capitalize this? It would also be good if you can do the same for `A Communications Center` so we are consistent with everything else.

Reply:
Will do I left it lowercase to match comms